packages/module/patternfly-docs/content/extensions/data-view/examples/Toolbar/Toolbar.md

```diff
@@ -43,7 +68,7 @@ The `useDataViewPagination` hook manages the pagination state of the data view.
 - optional `searchParams` object
 - optional `setSearchParams` function
```

Comment:
would capitalize the first word 

**Initial values:**
- `perPage` initial value
- Optional `page` initial value
- Optional `searchParams` object
- Optional `setSearchParams` function